http/headers/permissions-policy.json

```diff
@@ -0,0 +1,1682 @@
+{
+  "http": {
+    "headers": {
+      "Permissions-Policy": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Permissions-Policy",
+          "spec_url": "https://w3c.github.io/webappsec-permissions-policy/#permissions-policy-http-header-field",
+          "support": {
+            "chrome": [
+              {
+                "version_added": "88"
+              },
+              {
+                "alternative_name": "Feature-Policy",
+                "version_added": "60",
+                "version_removed": "87"
```

Comment:
Was support for the `Feature-Policy` header really removed, one release before its replacement even?